mqml/tests/test_conductresist.py

```diff
@@ -0,0 +1,78 @@
+import pytest
+
+from mqml.parameter.conductresist import ConductResist
+from qcodes.instrument.base import Instrument
+import numpy as np
+
+@pytest.fixture(autouse=True)
+def close_all_instruments():
+    """Makes sure that after startup and teardown all instruments are closed"""
+    Instrument.close_all()
+    yield
+    Instrument.close_all()
```

Comment:
Is this fixture needed. As far as I can see there are no instruments being created in these tests

Reply:
ConductResist is an Instrument.

Reply:
Sorry I take that back. I can see that ConductResist is an instrument. It is unfortunate that it is found in a folder called parameters when it is not a Parameter.

Reply:
No problem. The naming is not mature yet, you're right. I may perform some changes.